src/chrome/content/rules/Bitcoin-Contact.org.xml

```diff
@@ -1,35 +1,13 @@
 <!--
-	^: Expired 2012-11-17
-	www: Redirects to web1.srv0.eu
-
-
-	Insecure cookies are set for these hosts:
-
+	Mixed Content Blocking (MCB) tiggered:
 		- bitcoin-contact.org
-
 -->
-<ruleset name="Bitcoin-Contact.org" default_off="expired" platform="cacert">
-
-	<!--	Direct rewrites:
-				-->
+<ruleset name="Bitcoin-Contact.org" platform="mixedcontent">
```

Comment:
Is there any critical mixed content that breaks functionality? I only see blocked tracking scripts.